bpytop.py

```diff
@@ -1141,9 +1141,10 @@ def __init__(self, color: str, depth: str = "fg", default: bool = False):
 				else:
 					raise ValueError(f'RGB dec should be "0-255 0-255 0-255"')
 
-			ct = self.dec[0] + self.dec[1] + self.dec[2]
-			if ct > 255*3 or ct < 0:
-				raise ValueError(f'RGB values out of range: {color}')
+			for i in [0, 1, 2]:
```

Comment:
Minor, but something like this would be more pythonic and concise.
```
if not all(0 <= c <= 255 for c in self.dec):
	raise ValueError(f'One or more RGB values are out of range: {color}')
```

Reply:
Thank you for reviewing this. While I agree I'd like to hear first from @aristocratos acknowledging this is a bug before updating this PR.

Reply:
@GerbenWelter
No, it's definitively an oversight on my part. Good find.
And also agree that @UmarJ's edit looks neat :)